2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -407,26 +361,29 @@ flt_ratios <- ratios_matched %>%
 ```{r}
 # Create a wide table with unique ids as row names
 flt_ratios_wide <- flt_ratios %>% 
+  filter(matched=='Both spectrum matched') %>%
   select(Master.Protein.Accessions,
          Sequence, Modifications,
-         Replicate, Matched_NC, ratio) %>% 
-  pivot_wider(names_from = "Replicate", values_from = c("Matched_NC", "ratio")) %>% 
+         Replicate, ratio) %>% 
+  pivot_wider(names_from = "Replicate", values_from = "ratio") %>% 
```

Comment:
Due to above filter, no need to retain `Matched_NC` information, which impacts downstream lines...